lib/Cargo.toml

```diff
@@ -15,16 +15,24 @@ hacspec-lib = { version = "0.1.0-beta.1", default-features = false, optional = t
 edhoc-crypto = { path = "../crypto", default-features = false }
 edhoc-consts = { path = "../consts", default-features = false }
 edhoc-ead = { path = "../ead", default-features = false }
+panic-semihosting = { version = "0.6.0", features = ["exit"], optional = true }
+embedded-alloc = { version = "0.5.0", optional = true }
+
+[build-dependencies]
+cbindgen = "0.24.5"
 
 [features]
 default = [ "edhoc-ead/ead-none" ]
 hacspec-hacspec = ["hacspec-lib/std", "edhoc-hacspec", "edhoc-crypto/hacspec", "edhoc-consts/hacspec" ]
 hacspec-cc2538 = ["hacspec-lib/alloc", "edhoc-hacspec", "edhoc-crypto/cc2538", "edhoc-consts/hacspec" ] # FIXME: stubs
-hacspec-psa = ["hacspec-lib/alloc", "edhoc-hacspec", "edhoc-crypto/psa", "edhoc-consts/hacspec" ]
-hacspec-psa-baremetal = ["hacspec-psa", "edhoc-crypto/psa-baremetal", "edhoc-consts/hacspec" ]
+hacspec-psa = ["hacspec-lib/alloc", "edhoc-hacspec", "edhoc-crypto/psa", "edhoc-consts/hacspec", "embedded-alloc" ]
```

Comment:
I am missing why is it needed to add `embedded-alloc` as a dependency in this file?

Reply:
This is a remnant of when I was trying to have `staticlib` always in Cargo.toml (as opposed to only addding it when I want to build the static library, using `build_for_c.sh`).

It is not needed anymore, good catch.